docs/tutorials/solar_flares.ipynb

```diff
@@ -895,8 +873,29 @@
     "Or remove the existing binning, yielding a 1D-variable.\n",
     "\n",
     "```python\n",
-    "temporal = sc.bin(spatial, edges=[time_edges], erase=['x', 'y'])\n",
-    "temporal.bins.sum().plot(ylabel='number of flares')\n",
+    "temporal = spatial.bins.concat('x').bins.concat('y').bin(peak_time=200)\n",
```

Comment:
If there are no extra dims, would `spatial.flatten(to='dummy').bin(peak_time=200)` do the same and erase the `dummy` dim automatically?

Reply:
To answer my own question: It does not remove it.

Reply:
No. This doesn't erase `dummy`.